pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -101,7 +111,12 @@ func (h *handler) onRepo(key string, repo *catalog.ClusterRepo) (*catalog.Cluste
 		}
 		// webhook needs to be able to adopt the MutatingWebhookConfiguration which originally wasn't a part of the
 		// chart definition, but is now part of the chart definition
-		if err := h.manager.Ensure(chartDef.ReleaseNamespace, chartDef.ChartName, chartDef.MinVersionSetting.Get(), values, chartDef.ChartName == webhookChartName, installImageOverride); err != nil {
+		minVersion := chartDef.MinVersionSetting.Get()
+		exactVersion := chartDef.ExactVersionSetting.Get()
+		if chartDef.ChartName == webhookChartName && minVersion != "" {
+			exactVersion = ""
```

Comment:
Nit: Might be good to leave a comment here about how, for the webhook, we want the legacy parameter to take precedent.